cmd/template/cluster/flag.go

```diff
@@ -121,11 +122,21 @@ func (f *flag) Validate() error {
 	if !aws.ValidateRegion(f.Region) {
 		return microerror.Maskf(invalidFlagError, "--%s must be valid region name", flagRegion)
 	}
-	if f.MasterAZ == "" {
-		return microerror.Maskf(invalidFlagError, "--%s must not be empty", flagMasterAZ)
+
+	// AZ name(s)
+	if len(f.MasterAZ) != 1 && len(f.MasterAZ) != 3 {
```

Comment:
Shouldn't this be a or instead of and ?

```
if len(f.MasterAZ) != 1 || len(f.MasterAZ) != 3 {
```

Reply:
No, then the condition would be always true since 
`if (1 !=1) || (1 !=3)` or `if (3!=1) || (3!=3)`  is true

Reply:
Ah yeah sorry I was reading it wrongly, thanks 👍